geoh5py/objects/surveys/electromagnetics/base.py

```diff
@@ -531,9 +459,18 @@ def transmitters(self, transmitters: BaseEMSurvey):
         self.edit_em_metadata({"Transmitters": transmitters.uid})
 
     @property
-    @abstractmethod
-    def type(self):
-        """Survey element type"""
+    def omit_list(self) -> tuple:
```

Comment:
class attributes

Reply:
We unfortunatly need to access the private attribute on the subclass, otherwise if we try to call the private attribute on a method of the base class, we can the private of the base.

Reply:
I did not get it...

Reply:
Class attributes, if access by a property under the base class, returns the base class value, not the child one.